frappe/client.py

```diff
@@ -78,7 +78,13 @@ def get(doctype, name=None, filters=None, parent=None):
 	if frappe.is_table(doctype):
 		check_parent_permission(parent, doctype)
 
-	doc = frappe.get_doc(doctype, name or frappe.parse_json(filters))
+	if name:
+		doc = frappe.get_doc(doctype, name)
+	elif filters:
+		doc = frappe.get_doc(doctype, frappe.parse_json(filters))
```

Comment:
```suggestion
	elif filters is not None:
		doc = frappe.get_doc(doctype, frappe.parse_json(filters))
```

`filters` can be `{}`

Reply:
It can also be `''` like this very case. 

So `filters and isinstance(filters, str)` is better?

Reply:
`filters` will fail for an empty dict tho..